jme3-core/src/main/java/com/jme3/input/InputManager.java

```diff
@@ -150,6 +153,23 @@ public InputManager(MouseInput mouse, KeyInput keys, JoyInput joystick, TouchInp
         }
 
         firstTime = keys.getInputTimeNanos();
+
+        try {
+            Class mouseClass = Class.forName("org.lwjgl.input.Mouse");
+            Field initializedField = mouseClass.getDeclaredField("initialized");

```

Comment:
I think this class shouldn't know about "org.lwjgl.input.Mouse" or other implementations.